packages/xo-server/src/xapi/index.mjs

```diff
@@ -185,14 +185,24 @@ export default class Xapi extends XapiBase {
   //
   // If `force` is false and the evacuation failed, the host is re-
   // enabled and the error is thrown.
-  async clearHost({ $ref: ref }, force) {
-    await this.call('host.disable', ref)
+  async clearHost({ $ref: hostRef, $pool: pool }, force) {
+    await this.call('host.disable', hostRef)
 
+    const migrationNetworkId = pool.other_config['xo:migrationNetwork']
+    const migrationNetworkRef = migrationNetworkId && this.getObject(migrationNetworkId).$ref
     try {
-      await this.callAsync('host.evacuate', ref)
+      try {
+        await this.callAsync('host.evacuate', hostRef, migrationNetworkRef)
```

Comment:
What happens if
1.  supported and `undefined`?
2. unsupported and `undefined`?

Reply:
1. I can't test it
2. It triggers a `MESSAGE_PARAMETER_COUNT_MISMATCH` and thus a retry

So I'll change it since there's no point in passing `undefined` anyway.